### DISCUSSION TITLE
Show Operation and Html fields

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#918] [Q&A (Help)] [answered]
I've this method:
```
    protected function setupUpdateOperation()
    {
        $this->setupCreateOperation();

        $this->data["notifications"] = NotificationLanguages::where('notification_id', $this->crud->getCurrentEntryId())
        ->get()->toArray();

        CRUD::addField([
            'name' => 'content',
            'label' => trans('backpack::base.content'),
            'type' => 'translate_entity',
            'edit_mode' => true,
            'tab' => trans('backpack::base.content'),
        ]);
    }
```

in setupShowOperation() i'd like to show the field content with the list of all languages so i've created this:

```
    protected function setupShowOperation()
    {
        $this->setupListOperation();

        CRUD::addColumn([
            'name' => 'content_html', // Questo è un attributo virtuale, non un nome di colonna effettivo nel DB
            'label' => trans('backpack::base.content'),
            'type' => 'model_function',
            'function_name' => 'getContentHtmlAttribute', // La funzione che abbiamo definito nel modello
            'escaped' => false
        ]);

    }
```

and in my model:
```
    public function getContentHtmlAttribute() {
        $translations = $this->translations()->get();

        $htmlContent = '';
        foreach ($translations as $translation) {
            $languageName = strtoupper($translation->lang);
            $content = $translation->content;

            $htmlContent .= "<div><strong>{$languageName}:</strong><br /><p>{$content}</p></div><br /><br />";
        }

        return $htmlContent;
    }
```
Debugging code i see that $htmlContent return the right html:
<img width="693" alt="Screenshot 2024-04-04 alle 18 59 45" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/f8161197-2f99-4220-8fbd-9b246bdede44">

but on page i've this:
<img width="659" alt="pic-bkp-html-show" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/7d8484e8-2993-4f2a-bd3a-d6f3f4b3d764">

Is there a way to fix it and see this:>?

<img width="707" alt="Screenshot 2024-04-04 alle 18 59 34" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/03ed9362-c2f0-4daf-b0c7-cc6b64aa738f">

Thanks for help

**Comment by pxpm** [comment, marked as answer]
Hey @blondie63 

you probably need `escaped => false` and `limit => 999999` in your columns definition. 

Let me know if that helps. 

Cheers

**Reply by blondie63** [reply]
Perfect now !
Thanks
